pkg/services/alerting/riot/metrics.go

```diff
@@ -0,0 +1,65 @@
+package riot
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+type Metric struct {
+	Name     string `json:"name"`
+	Hostname string `json:"rfc460Hostname"`
+	Instance string `json:"instance"`
+
+	Value     interface{} `json:"value"`
+	Timestamp time.Time   `json:"rfc460Timestamp"`
+
+	Scope string `json:"rfc190Scope"`
+}
+
+// ConstructMetric generates a new Metric
+func ConstructMetric(name string, value interface{}) Metric {
+	hostname := os.Getenv("RC_HOSTNAME")
+
+	scope := fmt.Sprintf("%s.%s.%s", os.Getenv("RC_ENVIRONMENT"), os.Getenv("RC_DATACENTER"), os.Getenv("RC_SHARD"))
+
+	return Metric{
+		Name:      name,
+		Value:     value,
+		Hostname:  hostname,
+		Instance:  "grafana",
```

Comment:
NIT: should we use port number here to match the Instance suggestion we use for our other metrics methods (like hmp.metricsd)?